grafana-plugin/src/pages/settings/tabs/ChatOps/ChatOps.tsx

```diff
@@ -44,9 +45,17 @@ class ChatOpsPage extends React.Component<ChatOpsProps, ChatOpsState> {
 
   render() {
     const { activeTab } = this.state;
+    const { store } = this.props;
 
     return (
       <div className={cx('root')}>
+        {!store.hasFeature(AppFeature.Slack) && !store.hasFeature(AppFeature.Telegram) && (
+          <div>
+            <Alert severity="warning" title="No chatops found">
+              Chatops is disabled because no chat integration is configured. Please check docs for open source!
```

Comment:
Should we make a mention to "docs for open source"? This would also be shown on cloud instances. What about "Please check the documentation here" ("here" could be a link pointing people to the docs).

Reply:
@Matvey-Kuk these features are only for OSS, or am I mistaken?

Reply:
Added link for both telegram and slack as each separate page mentions the needed env variable to enable the given chatops feature